bazel_tools/grpc-haskell-core-upgrade.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/cbits/grpc_haskell.c b/cbits/grpc_haskell.c
+index ebbcbb1..236c73e 100644
+--- a/cbits/grpc_haskell.c
++++ b/cbits/grpc_haskell.c
+@@ -504,8 +504,7 @@ grpc_auth_metadata_processor* mk_auth_metadata_processor(
+ 
+ grpc_call_credentials* grpc_metadata_credentials_create_from_plugin_(
+   grpc_metadata_credentials_plugin* plugin){
+-
+-  return grpc_metadata_credentials_create_from_plugin(*plugin, NULL);
++  return grpc_metadata_credentials_create_from_plugin(*plugin, GRPC_PRIVACY_AND_INTEGRITY, NULL);
+ }
+ 
+ //This is a hack to work around GHC being unable to deal with raw struct params.
+diff --git a/src/Network/GRPC/Unsafe.chs b/src/Network/GRPC/Unsafe.chs
+index d11f533..6eb08b8 100644
+--- a/src/Network/GRPC/Unsafe.chs
++++ b/src/Network/GRPC/Unsafe.chs
+@@ -219,9 +219,6 @@ castPeek p = do
+     unTag `Tag'}
+   -> `()'#}
+ 
+-{#fun grpc_channel_ping as ^
+-  {`Channel', `CompletionQueue', unTag `Tag',unReserved `Reserved'} -> `()' #}
+-
+ {#fun grpc_channel_destroy as ^ {`Channel'} -> `()'#}
+ 
+ -- | Starts executing a batch of ops in the given 'OpArray'. Does not block.
+diff --git a/tests/LowLevelTests/Op.hs b/tests/LowLevelTests/Op.hs
+index 52b6d55..ff0751a 100644
+--- a/tests/LowLevelTests/Op.hs
```

Comment:
this doesn’t actually matter to us since it’s only a test but I feel more confident including the full patch including the fix for the tests here. Also makes it easier to upstream since we don’t have to figure out how to fix the tests again.